content/ri/using-redisinsight/api-get.md

```diff
@@ -0,0 +1,69 @@
+---
+Title: Adding Databases via GET URL
+date: 2021-01-11 00:00:00 +0000
+weight: 16
+categories: ["RI"]
+path: api/
+nextStep:
+    Title: Memory Analysis
+    href: /docs/features/memory-analysis/
+---
+
+If you want to automate adding a Redis database without filling the [database form]({{< relref "/ri/using-redisinsight/add-instance.md" >}}), you might want to add database via GET URL.
+
+Below is the documentation for the URL required to add databases.
+
+## Add Redis database
+
+Used to add Redis databases to RedisInsight.
+
+**URL** : `/add/`
+
+**Method** : `GET`
+
+
+### Query Parameters
+
+These are the required query parameters for any type of database.
```

Comment:
Is this the order they need to be in as well?

Reply:
No, the order of the parameters does not matter.